source/fresh-installation.rst

```diff
@@ -22,9 +22,9 @@ Create Conda Environments
 
 A puppet class should install conda into ``/opt/conda`` on machines designated
 ``*-srv*`` ("server") or ``*-ws*`` ("workstation") as soon as they are on the
```

Comment:
Is it worth mentioning the few exceptions that conda lives on ca or gpu machines?